.rubocop_todo.yml

```diff
@@ -0,0 +1,1029 @@
+    - 'app/helpers/application_helper.rb'
+    - 'app/models/ability.rb'
+    - 'app/models/hydrus/collection.rb'
+    - 'app/models/hydrus/solr_queryable.rb'
+
+# Offense count: 3
+# Cop supports --auto-correct.
+# Configuration parameters: AllowForAlignment, ForceEqualSignAlignment.
+Layout/ExtraSpacing:
+  Exclude:
+    - 'app/models/hydrus/generic_object_stuff.rb'
+    - 'app/models/hydrus/item.rb'
+
+# Offense count: 3
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles, IndentationWidth.
+# SupportedStyles: consistent, special_for_inner_method_call, special_for_inner_method_call_in_parentheses
+Layout/FirstParameterIndentation:
+  Exclude:
+    - 'app/models/hydrus/collection.rb'
+    - 'app/models/solr_document.rb'
+    - 'config/rsolr_certificate.rb'
+
+# Offense count: 2
+# Cop supports --auto-correct.
+# Configuration parameters: SupportedStyles, IndentationWidth.
+# SupportedStyles: special_inside_parentheses, consistent, align_brackets
+Layout/IndentArray:
+  EnforcedStyle: consistent
+
+# Offense count: 7
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles, IndentationWidth.
+# SupportedStyles: special_inside_parentheses, consistent, align_braces
+Layout/IndentHash:
+  Exclude:
+    - 'devel/dashboard_experiment.rb'
+    - 'lib/tasks/ci.rake'
+    - 'spec/features/collection_create_spec.rb'
+    - 'spec/models/hydrus/collection_spec.rb'
+
+# Offense count: 30
+# Cop supports --auto-correct.
+Layout/LeadingCommentSpace:
+  Enabled: false
+
+# Offense count: 3
+# Cop supports --auto-correct.
+Layout/MultilineBlockLayout:
+  Exclude:
+    - 'spec/models/hydrus/related_item_spec.rb'
+
+# Offense count: 19
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles.
+# SupportedStyles: symmetrical, new_line, same_line
+Layout/MultilineMethodCallBraceLayout:
+  Exclude:
+    - 'app/models/hydrus/admin_policy_object.rb'
+    - 'app/models/hydrus/contentable.rb'
+    - 'app/models/hydrus/generic_object_stuff.rb'
+    - 'app/models/hydrus/item.rb'
+    - 'remediations/run.rb'
+    - 'spec/controllers/hydrus_collections_controller_spec.rb'
+    - 'spec/controllers/hydrus_items_controller_spec.rb'
+    - 'spec/features/item_edit_spec.rb'
+
+# Offense count: 1
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles, IndentationWidth.
+# SupportedStyles: aligned, indented, indented_relative_to_receiver
+Layout/MultilineMethodCallIndentation:
+  Exclude:
+    - 'app/helpers/application_helper.rb'
+
+# Offense count: 2
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles, IndentationWidth.
+# SupportedStyles: aligned, indented
+Layout/MultilineOperationIndentation:
+  Exclude:
+    - 'app/models/hydrus/cant.rb'
+    - 'spec/helpers/hydrus_form_helper_spec.rb'
+
+# Offense count: 6
+# Cop supports --auto-correct.
+Layout/SpaceAfterNot:
```

Comment:
Ah, here's the rubocop rule that'd catch  e.g.`! var`. I think we should fix those up now too.